planning/motion_velocity_planner/autoware_motion_velocity_out_of_lane_module/src/filter_predicted_objects.cpp

```diff
@@ -154,6 +187,8 @@ autoware_perception_msgs::msg::PredictedObjects filter_predicted_objects(
         [](const auto & p) { return p.path.empty(); }),
       predicted_paths.end());
 
+    resample_predicted_paths(predicted_paths, filtered_object.shape);
+
     if (!filtered_object.kinematics.predicted_paths.empty())
       filtered_objects.objects.push_back(filtered_object);
   }
```

Comment:
Can put the call to resample function inside the if statement
```suggestion
    if (!filtered_object.kinematics.predicted_paths.empty())
      resample_predicted_paths(predicted_paths, filtered_object.shape);
      filtered_objects.objects.push_back(filtered_object);
  }
```